extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -31,32 +31,64 @@ if (context.RunTest) {
 			let server = await getAzureServer();
 			await VerifyOeNode(server, 3000, expectedNodeLabel);
 		});
-		test('context menu test', async function () {
-			let server = await getAzureServer();
-			await connectToServer(server, 3000);
-			let nodes = <azdata.objectexplorer.ObjectExplorerNode[]>await azdata.objectexplorer.getActiveConnectionNodes();
-			assert(nodes.length > 0, `Expecting at least one active connection, actual: ${nodes.length}`);
 
-			let index = nodes.findIndex(node => node.nodePath.includes(server.serverName));
-			assert(index !== -1, `Failed to find server: "${server.serverName}" in OE tree`);
+		test('Standard SQL DB context menu test', async function () {
+			let server = await getStandaloneServer();
 
-			let node = nodes[index];
-			let actions = await azdata.objectexplorer.getNodeActions(node.connectionId, node.nodePath);
-			let expectedActions;
+			let expectedActions: string[];
 
+			// Properties comes from the admin-tool-ext-win extension which is for Windows only, so the item won't show up on non-Win32 platforms
 			if (process.platform === 'win32') {
 				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler', 'Properties'];
 			} else {
 				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler'];
 			}
 
-			const expectedString = expectedActions.join(',');
-			const actualString = actions.join(',');
-			assert(expectedActions.length === actions.length && expectedString === actualString, `Expected actions: "${expectedString}", Actual actions: "${actualString}"`);
+			await verifyContextMenu(server, expectedActions);
+		});
+
+		test('BDC instance context menu test', async function () {
+			let server = await getBdcServer();
+
+			let expectedActions: string[];
+
+			// Properties comes from the admin-tool-ext-win extension which is for Windows only, so the item won't show up on non-Win32 platforms
+			if (process.platform === 'win32') {
+				expectedActions = ['Manage', 'New Query', 'Disconnect', 'Delete Connection', 'Refresh', 'New Notebook', 'Data-tier Application wizard', 'Launch Profiler', 'Properties'];
```

Comment:
hmm, does it make sense to define const for these menu items? they are duplicated 3 to 4 times.

Reply:
Possibly, going to leave them like this for now since it's likely the different server types will have more different options as time goes on so I see us moving back to this anyways but if we add more or something we can revisit that.